documents/final-submission-document/final-submission-document.tex

```diff
@@ -102,19 +102,24 @@
 \vspace*{1ex}
 
 %Goal
-The goal of this thesis is to engineer a specialised and platform independent XMPP-Grid Broker that configures existing XMPP Controllers (XMPP server). The XMPP-Grid Broker should enable administrators to configure XMPP-Grids in a usable and productive way.
-
-The focus of the application should be set on portability, extensibility and the security aspects in a production environment.\\
+The goal of this thesis is to design and implement an XMPP-Grid Broker to configure existing Controllers (XMPP server), focussing on portability, extensibility and the security aspects in a production environment.
+The  Broker application should enable administrators to configure XMPP-Grids in a usable and productive way.\\
 \vspace*{1ex}
 
 %Result
-We propose the architecture of an XMPP-Grid Broker to configure XMPP Controllers and present an implementation based on the Angular5 framework.
+Our proposed architecture earmarks a purely client-side application that communicates typically through a regular reverse proxy with the XMPP Controller over WebSockets or HTTP streams (BOSH).
```

Comment:
I find this sentence a bit long & hard to read.
Also, I'm not shure about the use of "earmarks". I assume you are? 😉

What do you think about splitting it up in two sentences like:

> Our proposed architecture earmarks a purely client-side application that communicates with the Controller over WebSockets or HTTP streams (BOSH).
> The Controller is typically protected by a reverse proxy, which also hosts our application.